app/main/helpers/frameworks.py

```diff
@@ -252,4 +252,4 @@ def get_status_for_multi_service_lot_and_service_type(
 def has_one_service_limit(lot_slug, framework_lots):
     for lot in framework_lots:
         if lot['slug'] == lot_slug:
-            return lot['one_service_limit']
+            return lot['oneServiceLimit']
```

Comment:
The API responds with both for now but the intention was to drop the old one some time soon. I've updated everything I could find in the supplier-frontend, were you using it somewhere else?